package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@finos/git-proxy",
-  "version": "1.3.4",
+  "version": "1.3.5-alpha.0",
```

Comment:
Are we removing this prior to NPM publish or want to publish under this namespace for testing purposes? Happy with either.

Reply:
I'll revert this line prior to merging. We can bundle this change up into a formal release.

From my limited testing on an internal NPM registry, this version would be selected over the latest "production" release (1.3.4) which is not what we intend for users.

Reply:
Got it, thanks @coopernetes ❤️